test/integration/restrictusers_test.go

```diff
@@ -16,12 +16,20 @@ import (
 )
 
 func TestRestrictUsers(t *testing.T) {
-	masterConfig, clusterAdminKubeConfig, err := testserver.StartTestMaster()
```

Comment:
I think we want to keep the changes in this file.

Reply:
we're removing this file, right? I'll add this back in for now, and remove the integration as a follow-up (to the follow-up from revert).